packages/modules/manage-versions/package.json

```diff
@@ -32,13 +32,15 @@
     "esm/**/*"
   ],
   "dependencies": {
-    "@itwin/itwinui-icons-react": "^2.2.0",
-    "@itwin/itwinui-react": "^2.12.18",
+    "@itwin/itwinui-icons-react": "^2.10.0",
+    "@itwin/itwinui-react": "^3.18.0",
```

Comment:
shouldn't itwinui be a peer?

Reply:
maybe, that would be a breaking change then :D I guess I was trying to avoid major release here :/